x/evm/types/types.go

```diff
@@ -724,7 +722,7 @@ func (c *CommandID) Unmarshal(data []byte) error {
 
 // ValidateBasic returns an error if the given command ID is invalid
 func (c CommandID) ValidateBasic() error {
-	if bytes.Equal(c[:], zeroID[:]) {
+	if c.IsZero() {
```

Comment:
Transfer ID of `0` isn't considered invalid (and was used up already). Since commandID for mint was generated from it, we shouldn't consider zero as invalid too.

Instead, the fix should just be to make this method a no-op I think

Reply:
I already found a couple of cases (granted, they where test setups), where we forgot to setup the command ID, so it was zero. In order to differentiate forgetting to set CommandID from it being zero on purpose I think it makes sense to just not allow the 0 case.

Reply:
Ok, but `0` has already been used for a valid command. E.g. 
`axelard q evm command ethereum 0000000000000000000000000000000000000000000000000000000000000000 --node https://axelartest-rpc.quickapi.com:443`

With this change, it'll be considered invalid now for a valid transfer which doesn't make sense. I agree that we should have prevented transfer ID from starting at `0` in the first place but seems too late to change that.

Reply:
That is only relevant for querying now, right? I can take that into consideration